alembic/versions/8e7282ae4840_updated_rtp_tables.py

```diff
@@ -0,0 +1,55 @@
+"""updated rtp tables
+
+Revision ID: 8e7282ae4840
+Revises: e66c069eb92b
+Create Date: 2017-08-13 18:06:40.892734+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '8e7282ae4840'
+down_revision = 'e66c069eb92b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table('host_status')
```

Comment:
That seems like a reasonable solution. In the meantime, I took the `drop_table` statement out of the `upgrade()` command, and the table addition out of the `downgrade()` command. Let me know if I should change it back.